.github/workflows/bank-api-ci.yml

```
@@ -42,4 +48,4 @@ jobs:
         run: dotnet tool install -g dotnet-format
       - name: Run Dotnet Format
         working-directory: src/backend/LoanHub.Backend
-        run: dotnet format --verify-no-changes LoanHub.Backend.sln
+        run: dotnet format --severity info --verify-no-changes LoanHub.Backend.sln
```

```suggestion
        run: dotnet format --severity info --verify-no-changes
```
Już jest tylko 1 solution w projekcie, więc nie potrzeba podawać explicite

---

faktycznie, dzieki